code/src/layouts/intro-banner/IntroBanner.scss

```diff
@@ -0,0 +1,24 @@
+@import "@design-system";
+
+.spa-banner-intro {
+  @include transition((background-image "slow" ease-in-out));
+
+  background-size: cover;
+  background-position: bottom;
+
+  &__wrapper {
+    max-width: 500px;
+    display: flex;
+    flex-direction: column;
+    gap: spacing(5);
+    padding-block: spacing(30);
+  }
+
+  &__heading {
+    line-height: 1.2;
+  }
+
+  &__button {
+    align-self: flex-start;
```

Comment:
Is it required here?

Reply:
You can see the result without this property. If we want to avoid it we need to create an additional div or add these styles. In my opinion, styles are a much more suitable solution
![image](https://github.com/idx-academy/spa-orders/assets/118522791/e77f017b-6d91-4430-af2c-daa18c3da288)